src/main/scala/mesosphere/marathon/core/launchqueue/impl/AppTaskLauncherActor.scala

```diff
@@ -221,41 +221,31 @@ private class AppTaskLauncherActor(
       log.info("Task launch for '{}' was accepted. {}", op.taskId, status)
   }
 
-  private[this] def receiveTaskStatusUpdate: Receive = {
-    case TaskStatusUpdate(_, taskId, status) =>
-      def handleTask(oldTask: Task) = {
-        oldTask.update(TaskStateOp.MesosUpdate(status, clock.now())) match {
-          case TaskStateChange.Update(updatedTask) =>
-            log.debug("updating status of {}", taskId)
-            tasksMap += taskId -> updatedTask
-
-          case TaskStateChange.Expunge =>
-            log.debug("{} finished", taskId)
-            removeTask(taskId)
-
-          case TaskStateChange.NoChange =>
-            log.debug("no change for {}", taskId)
-
-          case TaskStateChange.Failure(cause) =>
-            log.warning("Task state update failed for {}. Reason: {}", taskId, cause.getMessage)
-        }
+  private[this] def receiveTaskUpdate: Receive = {
+    case TaskUpdate(stateOp, stateChange) =>
+      log.info("received TaskUpdate")
```

Comment:
I think that I would get rid of this info log statement and promote the more specific log statements below to info if you want more visibility.